site/en/docs/extensions/upgrading/index.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: 'layouts/doc-post.njk'
+title: Upgrade to Manifest V3
+subhead: A guide to upgrading Manifest V2 extensions to Manifest V3 extensions.
+description: A guide to upgrading Manifest V2 extensions to Manifest V3 extensions.
+date: 2023-02-21
+---
+
+This section helps you upgrade an extension from Manifest V2 to Manifest V3, the newest version of the Chrome Extensions platform. A summary of the work is provided below with links to pages for each of the task areas. There is also a list of new extension features you should consider incorporating into your extension. Finally, there is a [migration checklist](/docs/extensions/mv3-migration-checklist) to help you keep track of your work.
+
+{% Aside %}
+Follow [What's new in Chrome Extensions](/docs/extensions/whatsnew/) to read about new Manifest V3 features as they become available.
+{% endAside %}
+
+## Migration summary
+
+Migration work is broadly divided into the categories below. The links provide details of each. To help you track your work, we've provided a checklist summarizing the contents of these documents. You can access the content via the checklist, or dive into the content. Both paths end with an upgraded extension. 
+
+* [Update the manifest](/docs/extensions/upgrade-to-mv3/update-the-manifest/)&mdash;The `manifest.json` must be specific to V3. Changes that can be made on their own are described in this section. Manifest changes related to code are described with the code changes they support.
+* [Upgrade to a service worker](/docs/extensions/upgrade-to-mv3/create-the-service-worker/)&mdash;A service worker replaces the extension's background or event page to ensure that background code stays off the main thread where it can hurt performance. This change also requires moving DOM, window, and certain extension API calls into offscreen documents. 
```

Comment:
@ali-spivak Curious about your thoughts on using the term "upgrade" when referring to the service worker